salesforce/10.11/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -3199,6 +3199,186 @@ Retrieves data from specified objects, whether or not they have been deleted.
 * SALESFORCE:NOT_FOUND
 * SALESFORCE:INVALID_INPUT
 
+[[queryResultList]]
+=== Query Result List
+`<salesforce:query-result-list>`
+
+Returns a list of result IDs for a specific batch. The Salesforce API documentation states that if the batch requires retries, there will be more than one <result> element in the output. This operation should be used with <<queryResultStreamById>> to obtain the results of the query.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Batch info a| <<BatchInfo>> |  BatchInfo to retrieve the results for. |  `#[payload]` | x
+| Content type a| Enumeration, one of:
+
+** XML
+** CSV
+** JSON
+** ZIP_XML
+** ZIP_CSV
+** ZIP_JSON |  Content type used at job creation. If it is not provided, the default value is used. |ContentType.XML  |
```

Comment:
"If it is not provided, the default value is used." This text is not necessary. This is true all the time.